package.json

```diff
@@ -16,8 +18,7 @@
     "start:website:blogOnly": "yarn workspace website start:blogOnly",
     "start:website:deployPreview": "cross-env NETLIFY=true CONTEXT='deploy-preview' yarn workspace website start",
     "examples:generate": "node generateExamples",
-    "build": "yarn build:packages && yarn build:website",
-    "build:packages": "lerna run build --no-private",
+    "build:packages": "yarn workspaces foreach --no-private run build",
```

Comment:
are you sure about this change?

Reply:
i think you are missing now

```json
"build": "yarn workspaces foreach run build",",
```

Reply:
This is implied. `yarn build` seems to work even without a custom script